packages/analysis-report/src/AnalysisResult.js

```diff
@@ -48,7 +53,7 @@ export const AnalysisResult = ( props ) => {
 				size="13px"
 			/>
 			<AnalysisResultText
```

Comment:
Can you use something like this to remove the dependency on `renderToString`?

```suggestion
			<AnalysisResultText suppressedText={ props.suppressedText }>
			    <BetaBadge />
			    <span dangerouslySetInnerHTML={ { __html: text } } />
			</AnalysisResultText>
```

Reply:
This works. Thanks!